kernel/idt.c

```diff
@@ -6,15 +6,15 @@
  * Definition of ISR (Interrupt Service Routines) to handle exceptions
  * and interrupts
  *
- * created: 2022/10/18 - xlmod <glafond-@student.42.fr>
- * updated: 2022/10/19 - lfalkau <lfalkau@student.42.fr>
+ * created: 2022/10/20 - mrxx0 <chcoutur@student.42.fr>
+ * updated: 2022/10/20 - mrxx0 <chcoutur@student.42.fr>
```

Comment:
Why changing the 'created' entry?

Reply:
We had an issue with the generated header.

Reply:
My mistake, the [autoheader plugin](https://github.com/lfalkau/autoheader) was updating the 9th line (hardcoded), so if the file description was longer that one line, bad things occured.
This is now fixed, so make sure to pull latest changes for the plugin